src/db.ts

```diff
@@ -43,6 +42,7 @@ export type odbcParams = {
   name: string
   user: string
   password: string
+  driverName : string
```

Comment:
is it optional? if so can you add a "?". Otherwise it will break backward compatibility